onecgiar-pr-server/src/api/results/results-validation-module/entities/validation.entity.ts

```diff
@@ -1,87 +1,93 @@
-import { Entity, PrimaryGeneratedColumn, Column, ManyToOne, JoinColumn, CreateDateColumn, UpdateDateColumn } from 'typeorm';
+import {
+  Entity,
+  PrimaryGeneratedColumn,
+  Column,
+  ManyToOne,
+  JoinColumn,
+  CreateDateColumn,
+  UpdateDateColumn,
+} from 'typeorm';
 import { Result } from '../../entities/result.entity';
 
 @Entity('validation')
-export class Validation{
+export class Validation {
+  @PrimaryGeneratedColumn()
+  id: number;
 
-    @PrimaryGeneratedColumn()
-    id: number;
+  @ManyToOne(() => Result, (r) => r.id)
+  @JoinColumn({
+    name: 'results_id',
+  })
+  results_id: number;
```

Comment:
Please try not to do this. Declare two variables: one just for the ID, and another one for the object. The way this currently is, does not make too much sense, as you are assigning a "Result" variable to a numeric property.